src/shared/buildspec/registry.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as filesystemUtilities from '../filesystemUtilities'
+import * as vscode from 'vscode'
+import * as yaml from 'js-yaml'
+import globals from '../extensionGlobals'
+import { WatchedFiles } from '../watchedFiles'
+import { isUntitledScheme, normalizeVSCodeUri } from '../utilities/vsCodeUtils'
+
+interface BuildSpecTemplate {
+    version?: string
+    phases?: any
```

Comment:
We're not passing a schema to `js-yaml` so this is the only accurate interface
```suggestion
    version?: unknown
    phases?: unknown
```